custom-connector-sdk/src/main/java/com/amazonaws/appflow/custom/connector/model/connectorconfiguration/auth/OAuth2MethodType.java

```diff
@@ -0,0 +1,8 @@
+package com.amazonaws.appflow.custom.connector.model.connectorconfiguration.auth;
+
+public enum OAuth2MethodType {
```

Comment:
Same, add java doc, on all public declarations in Builder SDK

Reply:
Ideally , we should add java doc for each individual enum as well, I see we followed this very well in other declarions.